tests/driver/runner_test.py

```diff
@@ -1750,6 +1750,26 @@ def test_create_ml_datafeed(self, es):
         r(es, params)
 
         es.xpack.ml.put_datafeed.assert_called_once_with(datafeed_id=params["datafeed-id"], body=params["body"])
+    @mock.patch("elasticsearch.Elasticsearch")
+    def test_create_ml_datafeed_fallback(self, es):
```

Comment:
I recommend we also create unit tests for the rest of the fallback enabled methods `DeleteMlDatafeed`, `StartMlDatafeed`, `StopMlDatafeed`, `CreateMlJob`, `DeleteMlJob`, `OpenMlJob`, `CloseMlJob`